plugin/storage/cassandra/schema/create.sh

```diff
@@ -10,7 +10,7 @@ function usage {
     >&2 echo "  DATACENTER         - datacenter name for network topology used in prod (optional in MODE=test)"
     >&2 echo "  TRACE_TTL          - time to live for trace data, in seconds (default: 172800, 2 days)"
     >&2 echo "  DEPENDENCIES_TTL   - time to live for dependencies data, in seconds (default: 0, no TTL)"
-    >&2 echo "  KEYSPACE           - keyspace (default: jaeger_v1_{datacenter})"
+    >&2 echo "  KEYSPACE           - keyspace (default: jaeger_v1_test)"
```

Comment:
it already defaults to test per L37.

Reply:
Schema name shoud never be dependent on value of datacenter. I tried executing cassandra-schema-job container without giving any keyspace value. I expect it to create `jaeger_v1_test` because that's the default value used by collector and query